core/alsp_src/tests/libtests/read_examp_toks.pro

```diff
@@ -0,0 +1,37 @@
+:-['./html_tokens.pro'].
```

Comment:
Is this consult necessary? Won't the library auto-load on first use?

An orthogonal issue: Removing this consult results in a false-positive test result (i.e. `make test` succeeds, but the test log shows a failure).

Reply:
Opps, ignore false-positive comment -- I didn't realize test results are only reported at end of test run.